### PR TITLE
Adding installation instructions

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

**Comment by mc12345678** [comment]
Actually, recommend items 1 and 2 to be swapped.  The code doesn't (or is intended to not) take any action if the "template" (meaning version specific) related items are not installed...  There may be some minor "inconsistency" there for example regarding the SBA attributes picture setting at the moment, though I think I found a sort of "history" for the selection of that default setting.  Might be able to make it smarter though.  Anyways, I drift from the point, 1 and 2 should be swapped otherwise there is a strong possible risk of website errors during installation that could otherwise be averted.

There's also the "larger" set of instructions that potteryhouse developed, but unfortunately has not been updated by the things I've changed.  It is only recently that the software is so much more ready to be submitted again as an updated copy and those instructions will need to be revisited to support that distribution.  But from a "simple" level, a five step process is a starter. :)

File: install.txt [hunk 1]
```diff
@@ -4,12 +4,12 @@ The directory corresponding to your cart's version will be called
...
-1) In YOUR_VERSION, change the directory names YOUR_TEMPLATE and 
-YOUR_CLASSIC_RESPONSIVE to correspond to what's being used in your cart.
+1) In includes, change the directory name YOUR_TEMPLATE 
+to correspond to what's being used in your cart.
 Then copy all the files in this directory to your cart.
 
-2) In includes, change the directory name YOUR_TEMPLATE 
-to correspond to what's being used in your cart.
+2) In YOUR_VERSION, change the directory names YOUR_TEMPLATE and 
+YOUR_CLASSIC_RESPONSIVE to correspond to what's being used in your cart.
```
Comment (mc12345678) [inline install.txt]: This (YOUR_CLASSIC_RESPONSIVE) was added in the event someone "copies" the classic_responsive directory to create their own template (in ZC 1.5.5+ only). Not sure I have a good suggestion of "words to use" but one could read this and try to change both directories to the same folder name. :(

File: install.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+b) If you are using 155 with a responsive template, 
+change the directory name YOUR_CLASSIC_RESPONSIVE to correspond to 
+what's being used in your cart, then remove YOUR_TEMPLATE.
+
+Then copy all the files in this directory to your cart.
+
+3) In admin, copy all files to your cart.
+
+4) Login to your cart's admin panel, and browse to
```
Comment (mc12345678) [inline install.txt:25]: Unfortunately at this point the configuration option for SBA has not been loaded. Making it available requires entering the filename on the browser. I haven't dug in to determine if this is on purpose or just wasn't coded to happen. It has been documented though in potteryhouse's instructions found under his github repo section of https://github.com/potteryhouse/stock_by_attribute_1.5.X_Instructions that I have also forked to my account.

File: install.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+what's being used in your cart, then remove YOUR_TEMPLATE.
+
+Then copy all the files in this directory to your cart.
+
+3) In admin, copy all files to your cart.
+
+4) Login to your cart's admin panel, and browse to 
+Configuration->Products with Attributes Stock (aka SBA) Setup. 
+
+5) Select Full/Upgrade DB Install from the dropdown and press the
```
Comment (mc12345678) [inline install.txt:28]: Might also advise that running this on an already installed version of SBA will maintain the existing SBA product data. Ie. It does not delete the table of product quantities or delete the quantities and variants within it.

**Comment by scottcwilson** [comment]
Hey, can you go ahead and write an install instruction document since you seem to know all the subtleties involved here?  I'd rather not keep going back and forth.  Why not take my PR and then add fixes as you see fit.

**Pull request closed without merging**